### DISCUSSION TITLE
Docs on textual.textualize.io can be ahead of latest release

### DISCUSSION THREAD

**Discussion by davetapley** [Textualize/textual#3463] [Ideas]
I just got very confused because VSCode couldn't find `@on(TextArea.Changed)` per [here](https://textual.textualize.io/widgets/text_area/#textual.widgets._text_area.TextArea.Changed):

<img width="1378" alt="Screen Shot 2023-10-04 at 9 26 19 PM" src="https://github.com/Textualize/textual/assets/48232/74b957d2-1942-4e52-8980-020f0f1fff42">

I verified I was on `v0.38.1`, as shown in the header ↗️ 

---

Turns out, I _presume_, those docs are from 3 day old:
- #3442 

but that hasn't actually been released yet  😬

**Comment by davep** [comment]
Aye, this is a small unfortunate side-effect of the blog being mixed in with the rest of the docs; so on some rare occasions, where a new blog post needs to be published ahead of a release of Textual, the newer Textual docs can "leak". It's something we'd eventually like to tidy up (I think we have an issue floating around somewhere for this -- could be on the internal board).

The one upside is that it does remind us some folk actually do read the docs! ;-)
